lib/MongoDB/Op/_Aggregate.pm

```diff
@@ -61,6 +63,17 @@ has has_out => (
     isa      => Bool,
 );
 
+has maxAwaitTimeMS => (
+    is       => 'rw',
+    isa      => Num,
+);
+
+has cursorType => (
```

Comment:
I think this is unnecessary.  For aggregation, if there's maxAwaitTimeMS, then we can conclude that it's a tailable await.